Extension/src/LanguageServer/client.ts

```diff
@@ -3747,6 +3737,15 @@ export class DefaultClient implements Client {
     public setReferencesCommandMode(mode: refs.ReferencesCommandMode): void {
         this.model.referencesCommandMode.Value = mode;
     }
+
+    public async addTrustedCompiler(path: string): Promise<void> {
+        // Detect duplicate paths or invalid paths.
+        if (trustedCompilerPaths.includes(path) || path === null || path === undefined) {
+            return Promise.resolve();
```

Comment:
As an `async` function, you could just return instead of resolving a promise.

Reply:
Ah, okay thanks.

Reply:
Fixed